### PR TITLE
sp, and wording

### PR DESCRIPTION
**Pull request by benaadams**

### CODE REVIEW

File: docs/Span.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ We need to enable the existing array bounds check optimizations for Span – in
 Span\<T\> will support reinterpret cast conversions to Span\<byte\>. It will also support unsafe casts between arbitrary primitive types. The reason for this limitation is that some processors don’t support efficient unaligned memory access.
 
 ##Platform Support Plans
-We want to ship Span\<T\> as a nugget fat package available for .NET Standard 1.1 and above. Runtimes that support by-ref fields and returns will get the fast two filed span. Other runtimes will get the slower three-field span. 
+We want to ship Span\<T\> as a nuget heritage package available for .NET Standard 1.1 and above. Runtimes that support by-ref fields and returns will get the fast two filed span. Other runtimes will get the slower three-field span.
```
Comment (davidfowl) [inline docs/Span.md]: heritage package isn't a thing lol 😄

Comment (benaadams) [inline docs/Span.md]: For heritage packages for heritage runtime versions 😉

**Comment by KrzysztofCwalina** [comment]
What are they really called?

**Comment by davidfowl** [comment]
There's no name for this, @benaadams is trying to coin a term 😉

**Comment by KrzysztofCwalina** [comment]
I would call them "adaptive packages"

File: docs/Span.md (hunk 1 above)
Comment (KrzysztofCwalina) [inline docs/Span.md]: @weshaggard, @ericstj, what do we call these packages?

**Comment by KrzysztofCwalina** [comment]
Closing for now, though it would be great to find an official name.
@terrajobst

File: docs/Span.md (hunk 1 above)
Comment (terrajobst) [inline docs/Span.md]: We informally call them fat packages. Publicly, there is no name for that and I'm convinced we need one. The nature of packages allows having multiple assets per platform.

**Comment by terrajobst** [comment]
See comment above

File: docs/Span.md (hunk 1 above)
Comment (benaadams) [inline docs/Span.md]: Ah like a bundle that includes x86, x64 etc all in one; portable but with baggage

**Pull request closed without merging**